ui/src/environments/environment.prod.ts

```diff
@@ -4,7 +4,7 @@ const {orbApi: {apiUrl, version, urlKeys, servicesUrls}} = defaultEnvironment;
 
 export const environment = {
   production: true,
-
+  GTMID: 'GTM-K44NMJP',
```

Comment:
@gpazuch point of interest here, need to check with @weyrick if these ids should be somehow hidden and not committed?

Reply:
we discussed that it should be okay